aptos-move/framework/move-stdlib/sources/vector.move

```diff
@@ -9,10 +9,12 @@
 /// Move functions here because many have loops, requiring loop invariants to prove, and
 /// the return on investment didn't seem worth it for these simple functions.
 module std::vector {
```

Comment:
Per discussion, can you add vector::insert from https://github.com/move-language/move/blame/74a7436617f6488717c87979968369c0d281ea86/language/move-stdlib/sources/vector.move#L149 as well?